docs/Displaying a User Interface/Popup Keyboards/index.md

```diff
@@ -202,14 +202,66 @@ KeyboardListener keyboardListener = new KeyboardListener() {
 ```
 !@
 
-@![iOS, android, javaSE, javaEE]
+@![javascript]
+```javascript
+const keyboardListener = new KeyboardListener()
+    .setOnUserDidSubmitInput((event) => {
+        switch (event) {
+            case KeyboardEvent.ENTRY_VOICE:
+                // The user decided to start voice input, you should start an AudioPassThru session if supported
+                break;
+            case KeyboardEvent.ENTRY_SUBMITTED:
+                // The user submitted some text with the keyboard
+                break;
+            default:
+                break;
+        }
+    })
+    .setOnKeyboardDidAbortWithReason((event) => {
+        switch (event) {
+            case KeyboardEvent.ENTRY_CANCELLED:
+                // The user cancelled the keyboard interaction
+                break;
+            case KeyboardEvent.ENTRY_ABORTED:
+                // The system aborted the keyboard interaction
+                break;
+            default:
+                break;
+        }
+    })
+    .setUpdateAutocompleteWithInput((currentInputText, keyboardAutocompleteCompletionListener) => {
+        // Check the input text and return a list of autocomplete results
+        keyboardAutocompleteCompletionListener(updatedAutoCompleteList);
+    })
+    .setUpdateCharacterSetWithInput((currentInputText, keyboardCharacterSetCompletionListener) => {
+        // Check the input text and return a set of characters to allow the user to enter
+    })
+    .setOnKeyboardDidSendEvent((event, currentInputText) => {
+        // This is sent upon every event, such as keypresses, cancellations, and aborting
+    })
+    .setOnKeyboardDidUpdateInputMask((event) => {
```

Comment:
This method doesn't exist. Right now it's `setOnKeyboardInputMaskHasChanged`. Change the doc or change the KeyboardListener setter name?